tools/test/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@bazel_skylib//rules:diff_test.bzl", "diff_test")
+load("@aspect_bazel_lib//lib:jq.bzl", "jq")
+
+jq(
+    name = "final_package_json",
+    # buildifier: do not sort
```

Comment:
nit: I'd mention that order matters here explicitly in a comment above the `# buildifier: do not sort` line since the order these are passed to `jq` is load bearing

Reply:
Good call. I should also add this in it's actual usage in `defaults.bzl`. It doesn't get auto-get sorted there, but someone could change some variable names later on.

Reply:
You read my mind. Just left a comment there as well.